src/index.mts

```diff
@@ -107,9 +109,14 @@ for (let index = 0; index < accounts.length; index++) {
       url: `https://app.circleci.com/settings/project/${repo.slug}/environment-variables`,
       variables: await getPaginatedData<CircleCIProjectVariable>(
         CIRCLE_TOKEN,
-        repo.id,
+        repo.slug,
```

Comment:
im confused why this shows the change but this was fixed here, so I think a rebase should fix this: https://github.com/CircleCI-Public/CircleCI-Env-Inspector/commit/c53b342e72a4e1e1fc4af70f34c7ea7496025af3

Reply:
i had made the changes already but didn't make the pull request. i rebased too. should be fixed now though